shared-helpers/src/locales/general.json

```diff
@@ -608,6 +608,163 @@
   "footer.mtcLogo": "Association of Bay Area Governments - Metropolitan Transportation Commission Logo",
   "footer.twitterLogo": "Twitter Logo",
   "footer.youtubeLogo": "YouTube Logo",
+  "help.getStarted.howDoIGetStarted": "How Do I Get Started?",
```

Comment:
All this text will be reviewed by the Content team, so they will check if it matches their writings.  No need for you to review it all, unless you want to,.